sections/featured-blog.liquid

```diff
@@ -265,7 +281,7 @@
     {
       "name": "t:sections.featured-blog.presets.name",
       "settings": {
-        "blog": "News"
+        "blog": "test"
```

Comment:
Is there a reason we opted to use a fake blog name here instead just removing the setting preset altogether?